vsm

```diff
@@ -103,6 +104,10 @@ class State(object):
     def got_signal(self, signal, value):
         vars(self.variables)[signal] = value
 
+        od = collections.OrderedDict(sorted(vars(self.variables).items()))
+        for k, v in od.items(): 
+            print("{} = {}".format(k, v))
```

Comment:
It's better if you can be able to see you are seeing state, otherwise how will you differentiate it from signals emitted? How about you print something like this:
```
State = {
  transmission = false
  phone_call = 'inactive'
}
```
You can do that by doing something like this:

```
print('State = {')
for k, v in sorted(vars(self.variables).items()):
    print("  {} = {}".format(k, v))
print('}')
```

That should do it =)